tests/test_wave_01.py

```diff
@@ -118,13 +118,10 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    assert [{'genre': 'Horror', 'rating': 3.5, 'title': 'It Came from the Stack Trace'}] == updated_data["watched"]
```

Comment:
This works totally correctly but it is a bit more common to assert the result first, e.g. 
```python
assert <RESULT> == <WHAT YOU EXPECT>
```